frontend/app/dev-dashboard/projects/[id]/page.js

```diff
@@ -0,0 +1,136 @@
+          { label: project.title, href: `/dev-dashboard/projects/${id}` },
+        ]}
+      />
+      <div className='flow-root md:mx-6 mx-0 bg-white rounded-xl shadow-lg p-6 max-w-full overflow-auto'>
+        <div className='md:flex-row flex-col md:justify-between flex md:items-center mb-2'>
+          <div className='flex-col flex mb-4'>
+            <div className='md:text-3xl text-lg font-semibold text-primary-blue-dark '>{project.title}</div>
+            <div
+              className='flex md:mt-4 mt-2 items-center space-x-2 text-sm font-semibold rounded-full max-w-fit md:px-3 px-1.5 py-1 whitespace-nowrap bg-primary-neutral-light'
+              style={{ backgroundColor: statusColor }}
+            >
+              {getStatusIcon(project.status)}
+              <span className='text-gray-700'>{capitalizeStatus(project.status)}</span>
+            </div>
+          </div>
+
+          <div className='flex md:justify-end items-center justify-evenly gap-3 mb-2 text-xs'>
+            <div className='flex items-center gap-1'>
+              <p className='inline md:hidden text-sm'>Print</p>
+              <PrintInvoicingButton projectID={project.id} />
+            </div>
+
+            <div className='flex items-center gap-1'>
+              <p className='inline md:hidden text-sm'>Edit</p>
+              <EditButton projectID={project.id} />
+            </div>
+
+            <div className='flex items-center gap-1'>
+              <p className='inline md:hidden text-sm'>Delete</p>
+              <DeleteButton projectID={project.id} setProjects={setProject} />
+            </div>
+          </div>
+        </div>
+
+        <div className='text-sm md:text-base grid grid-cols-1 md:grid-cols-2 md:gap-6 gap-0 bg-white text-gray-800'>
+          <div>
+            <p className='flex items-center mb-4'>
+              <PersonOutlineOutlinedIcon className='mr-3 text-primary-blue-dark text-xl md:text-2xl' />
+              <strong className='mr-2 text-base'>Client:</strong> {project.client?.name}
+            </p>
+            <p className='flex items-center mb-4'>
+              <EuroOutlinedIcon className='mr-3 text-primary-blue-dark text-xl md:text-2xl' />
+              <strong className='mr-2 text-base'>Budget:</strong> ${project.budget}
+            </p>
+          </div>
+
+          <div>
+            <p className='flex items-center mb-4'>
+              <CalendarTodayOutlinedIcon className='mr-3 text-primary-blue-dark text-xl md:text-2xl' />
+              <strong className='mr-2 text-base'>Start Date:</strong> {project.startDate}
+            </p>
+            <p className='flex items-center mb-4'>
+              <CalendarTodayOutlinedIcon className='mr-3 text-primary-blue-dark text-xl md:text-2xl' />
+              <strong className='mr-2 text-base'>Deadline:</strong> {project.deadline}
+            </p>
+          </div>
+        </div>
+
+        {/* {project.demoUrl && ( */}
```

Comment:
👀

Reply:
That's because we currently donot have demoUrl at database, so I just  use a fake image here to simulate the demo area.😊